### DISCUSSION TITLE
What is this % symbol in this piece of code?

### DISCUSSION THREAD

**Discussion by nerffan1** [FLOSIC/PublicRelease_2020#33] [Q&A]
Hello,

I found this code snippet in **nbodrv.f** (line 207-212):

```fortran
      DO INUC=1,NUM_ATMS
        CALL GET_LETTER(XMOL_LIST(INUC)%ANUM,LETTER)
        WRITE(99,5100) XMOL_LIST(INUC)%ANUM,XMOL_LIST(INUC)%ANUM,
     %            XMOL_LIST(INUC)%RX*AU2ANG,XMOL_LIST(INUC)%RY*AU2ANG, 
     %            XMOL_LIST(INUC)%RZ*AU2ANG
      END DO 
```

I can't find any sources on the usage of **%** as a line continuation symbol. Is this something from older Fortran? Or something else?

Thank You

**Comment by jacks1ka** [comment]
Hi Angel-
  I think you can use any character as long as its in column 6.  Write a little program and check!

Alan

From: Angel-Emilio Villegas Sanchez ***@***.***>
Date: Sunday, August 6, 2023 at 6:06 PM
To: FLOSIC/PublicRelease_2020 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [External] [FLOSIC/PublicRelease_2020] What is this % symbol in this piece of code? (Discussion #33)

Hello,

I found this code snippet in nbodrv.f (line 207-212):

      DO INUC=1,NUM_ATMS

        CALL GET_LETTER(XMOL_LIST(INUC)%ANUM,LETTER)

        WRITE(99,5100) XMOL_LIST(INUC)%ANUM,XMOL_LIST(INUC)%ANUM,

     %            XMOL_LIST(INUC)%RX*AU2ANG,XMOL_LIST(INUC)%RY*AU2ANG,

     %            XMOL_LIST(INUC)%RZ*AU2ANG

      END DO

I can't find any sources on the usage of % as a line continuation symbol. Is this something from older Fortran? Or something else?

Thank You
